src/lib/grid/container.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input, ViewEncapsulation, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+    selector: 'apto-container',
```

Comment:
We refer to this as a grid a lot, why not `apto-grid` rather than `apto-container`?

Reply:
I did this because a container can be used outside of the row/column component.